src/app/components/Headings/index.test.jsx

```diff
@@ -11,6 +11,23 @@ describe('Headline component', () => {
 describe('SubHeading component', () => {
   shouldMatchSnapshot(
     'should render correctly',
-    <SubHeading>This is a SubHeading</SubHeading>,
+    <SubHeading text="This is a SubHeading">This is a SubHeading</SubHeading>,
+  );
+
+  shouldMatchSnapshot(
+    'should render without quotes',
```

Comment:
The test description here makes it seem like the text itself should not render quotes - but it is only the id that renders without quotes. Same with the ones below.